_data/mentors.yml

```diff
@@ -174,3 +174,9 @@
   image: JasonWong.jpg
   twitter: codethejason
   bio: A cryptocurrency enthusiast.
+  
+- name: Hon Nguyen
+  github: vanhonit
+  image: hon.png
+  twitter: vanhonit
+  bio: works on linux server and management.
```

Comment:
We all have family issues...throwing your github and twitter as if it was a mentor's and thinking you'd get away with it...that's not even an excuse. I've moved past it.